cloudformation/membership-attribute-service.yaml

```diff
@@ -283,6 +283,16 @@ Resources:
       - MetricValue: 1
         MetricNamespace: "members-data-api"
         MetricName: !Sub "${Stage} - Default Payment Method set to nothing"
+  MembersDataHttpQueuesFullMetricFilter:
+    Type: AWS::Logs::MetricFilter
+    DependsOn: MembersDataApiLogGroup
+    Properties:
+      FilterPattern: "\"Max wait queue limit of 256 reached, not scheduling.\""
```

Comment:
Is this condition going to be a bit brittle, or is it the only feasible thing to measure?

Reply:
Currently it is the only message visible to us and only happens under extreme load. We have more work for further investigation - It is just more specific than monitoring a number of 4xx which happens on every busy news day